rasa/shared/constants.py

```diff
@@ -99,3 +99,34 @@
 
 RESPONSE_CONDITION = "condition"
 CHANNEL = "channel"
+
+CARRY_OVER_SLOTS_KEY = "carry_over_slots_to_new_session"
```

Comment:
Oh yes, I placed them here to avoid circular import in the first commits when I created that `domain_loading.py` utils module, I can move them back to `domain.py`.